### PR TITLE
Brand pivot & face lift

### PR DESCRIPTION
**Pull request by palmenhq**
I know it's a bit huge, sorry 🙈 actually not sorry, this is amazing.

Also enables `esModuleInterop` (like we [use in web-next](https://github.com/HedvigInsurance/web-next/pull/389)) and that's partially why so many files changed

Screenshot 2020-06-01 at 08 46 43

Screenshot 2020-06-01 at 08 46 09

Screenshot 2020-06-01 at 08 46 24

### CODE REVIEW

File: shared/hedvig-ui/button.tsx [hunk 1]
```diff
@@ -28,41 +29,48 @@ export interface ButtonProps {
...
+) => Record<
   NonNullable<ButtonProps['variation']>,
   { foreground: string; background: string; highlighted: string }
-> = {
+> = (theme = lightTheme) => ({
   default: {
-    foreground: colorsV2.white,
-    background: colorsV2.darkgray,
-    highlighted: colorsV2.gray,
+    foreground: theme.foreground,
```
Comment (sampettersson) [inline shared/hedvig-ui/button.tsx:39]: Sweeet

File: shared/hedvig-ui/button.tsx [hunk 2]
```diff
@@ -89,35 +97,39 @@ export const Button = styled('button')<ButtonProps>(
...
+      alignItems: 'center',
+      justifyContent: 'center',
+      whiteSpace: 'nowrap',
+      fontSize: buttonSizeMap[size].fontSize,
+      fontFamily: 'inherit',
+      padding: buttonSizeMap[size].padding,
+      width: fullWidth ? '100%' : 'auto',
+      minWidth: halfWidth ? '50%' : 'auto',
+      background: basic ? 'transparent' : colors.background,
+      color: (basic ? colors.background : colors.foreground) + ' !important',
```
Comment (sampettersson) [inline shared/hedvig-ui/button.tsx:114]: `!important` ⚠️  something overwriting it? 🙈

File: src/App.js [hunk 3]
```diff
@@ -23,93 +33,149 @@ if (process.env.NODE_ENV !== 'production') {
...
-                    />
-                    <Route
-                      path="/ticket_history/:id"
-                      render={(routeProps) => (
-                        <Routes.TicketHistoryPageRoute
-                          {...routeProps}
-                          store={store}
+      <DarkmodeContext.Provider
+        value={{
+          isDarkmode: this.state.isDarkmode,
```
Comment (sampettersson) [inline src/App.js:84]: Dunno if you do it, but would be cool if this respected `prefers-color-scheme`, one can use matchMedia to figure out in javascript I think :)

File: shared/hedvig-ui/button.tsx (hunk 2 above)
Comment (palmenhq) [reply shared/hedvig-ui/button.tsx:114]: semantic ui sometimes has very high specificity :( its either `!important` or some `&&&`s :(

File: src/App.js (hunk 3 above)
Comment (palmenhq) [reply src/App.js:84]: Would be 🆒 ! Maybe a future improvement? :)

File: src/components/dashboard/index.tsx [hunk 4]
```diff
@@ -1,43 +1,41 @@
-import { colors } from '@hedviginsurance/brand'
+import { MainHeadline } from 'hedvig-ui/typography'
```
Comment (gustaveen) [inline src/components/dashboard/index.tsx:1]: Not related to this PR, but would be nice to just have a Headline component and use the `as` prop if you want to render it as different headings :) 
```
<Heading as="h1" />
```

File: src/components/member/tabs/quote-tab/quote-list-item.tsx [hunk 5]
```diff
@@ -18,18 +17,18 @@ import { QuoteActivation } from './quote-activation'
 import { QuoteContractCreation } from './quote-contract-creation'
 import { QuoteModification } from './quote-modification'
 
-const OuterWrapper = styled('div')({
+const OuterWrapper = styled('div')(({}) => ({
   ':not(:last-child)': {
-    borderBottom: '1px solid ' + colorsV2.lightgray,
+    // borderBottom: '1px solid ' + theme.border,
```
Comment (gustaveen) [inline src/components/member/tabs/quote-tab/quote-list-item.tsx:22]: Remove? :)

File: src/components/shared/navigation/sidebar/VerticalMenu.tsx [hunk 6]
```diff
@@ -1,180 +1,345 @@
...
+                </MenuItem>
+                <MenuItem to="/tools">
+                  <Tools />
+                  <MenuText>Tools</MenuText>
+                </MenuItem>
+              </MenuGroup>
+
+              <MenuGroup>
+                <MenuItemExternalLink
+                  href="https://backoffice.trustly.com/?Locale=en_GB#/tab_orders"
```
Comment (AlexSJ1990) [inline src/components/shared/navigation/sidebar/VerticalMenu.tsx:296]: 😍

File: src/components/shared/navigation/sidebar/VerticalMenu.tsx [hunk 7]
```diff
@@ -1,180 +1,345 @@
...
+              </MenuGroup>
+            </Menu>
+
+            <BottomSection>
+              <DarkmodeSwitch>
+                <DarkmodeInnerSwitch
+                  type="checkbox"
+                  onChange={() => toggleDarkmode()}
+                />
+                {isDarkmode ? '🌞' : '🌚'}
```
Comment (AlexSJ1990) [inline src/components/shared/navigation/sidebar/VerticalMenu.tsx:322]: So great!!

**Comment by palmenhq** [comment]
@gustaveen thank you ❤️  actually we've talked about getting rid of material. Because i agree, we shouldn't have two UI libs. Semantic is used way more extensively, plus its a bit easier to wrap your head arond how it works imo so thats been the reasoning. But maybe we should bring that discussion up again idk 🤷‍♀️

**Pull request merged**